drivers/pcie/host/core.c

```diff
@@ -0,0 +1,215 @@
+/*
```

Comment:
same as for the file name, controller.c would be better no?

Reply:
Yes